backend/src/test/java/gov/cdc/usds/simplereport/service/ApiUserServiceTest.java

```diff
@@ -2,46 +2,40 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import gov.cdc.usds.simplereport.api.pxp.CurrentPatientContextHolder;
 import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
 import gov.cdc.usds.simplereport.service.model.UserInfo;
 import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration.WithSimpleReportOrgAdminUser;
 import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration.WithSimpleReportSiteAdminUser;
-import gov.cdc.usds.simplereport.test_util.TestDataFactory;
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.EnumSet;
 import java.util.List;
-import org.junit.jupiter.api.BeforeEach;
+import java.util.Set;
+import java.util.function.BiConsumer;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
 
 class ApiUserServiceTest extends BaseServiceTest<ApiUserService> {
-  @Autowired private TestDataFactory _dataFactory;
-
-  @Autowired private CurrentPatientContextHolder _currentPatientContextHolder;
-
-  @BeforeEach
-  void setupData() {
-    initSampleData();
-  }
 
   // The next several retrieval tests expect the demo users as they are defined in the
   // no-security and no-okta-mgmt profiles
   @Test
   @WithSimpleReportOrgAdminUser
   void getUsersInCurrentOrg_adminUser_success() {
+    initSampleData();
     List<UserInfo> users = _service.getUsersInCurrentOrg();
     Collections.sort(users, new UserInfoEmailComparator());
-    assertEquals(users.size(), 4);
-    assertEquals(users.get(0).getEmail(), "ben@sample.com");
-    assertEquals(users.get(0).getRoles(), List.of(OrganizationRole.USER));
-    assertEquals(users.get(1).getEmail(), "bob@sample.com");
-    assertEquals(users.get(1).getRoles(), List.of(OrganizationRole.USER, OrganizationRole.ADMIN));
-    assertEquals(users.get(2).getEmail(), "jamar@sample.com");
-    assertEquals(
-        users.get(2).getRoles(), List.of(OrganizationRole.ENTRY_ONLY, OrganizationRole.USER));
-    assertEquals(users.get(3).getEmail(), "sarah@sample.com");
-    assertEquals(users.get(3).getRoles(), List.of(OrganizationRole.USER, OrganizationRole.ADMIN));
+    BiConsumer<UserInfo, Set<OrganizationRole>> roleCheck =
```

Comment:
this seems like a more convoluted way to define a function of 2 variables that runs an assert statement?

Reply:
I don't think it's all that convoluted, other than the type declaration (which, yeah, Java). Potentially more readable to make it a full-fledged method, but it's only used three times and this and those lines all fit on the same editor screen right now...